.circleci/config.yml

```diff
@@ -84,4 +153,20 @@ jobs:
             dbt run --target bigquery --full-refresh
             dbt test --target bigquery
             dbt run --vars '{shopify__using_order_adjustment: false, shopify__using_order_line_refund: false, shopify__using_refund: false}' --target bigquery --full-refresh
-            dbt test --target bigquery
\ No newline at end of file
+            dbt test --target bigquery
+
+workflows:
+  version: 2
+  integration-tests:
+    jobs:
+      - test
+      - generate-docs:
+          requires:
+            - test
+          filters:
+            branches:
+              only:
+                - master
+                - main
```

Comment:
To make sure I am understanding this workflow
- Every time a commit is pushed the `test` job will run
- the `generate_docs` job will only be ran if the commit is against the `master` or `main` branches. 

Is this correct?